### PR TITLE
Make extraction script compatible with python 3

### PR DESCRIPTION
**Pull request by lgbaldoni**
Cf. https://github.com/imirkin/re-vp2/pull/5

### CODE REVIEW

File: extract_firmware.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```
Comment (imirkin) [inline extract_firmware.py]: Is this necessary? If there's no py2, python will be python3, right? And the script should work either way...

File: extract_firmware.py [hunk 2]
```diff
@@ -405,27 +411,20 @@ def decompress(prefix, start, s):
...
-    for start, end in itertools.izip(gzip_starts,
-                                     gzip_starts[1:] + [len(kernel)]):
+    for start, end in zip(gzip_starts, gzip_starts[1:] + [len(kernel)]):
         if decompress(names[idx], start, kernel[start:end]):
             idx += 1
     if idx != len(names):
-        print "Unexpected quantity of archives in blob, graph fw likely wrong."
+        print("Unexpected quantity of archives in blob, graph fw likely wrong.")
 else:
-    print "Unknown PGRAPH archive order in this version."
```
Comment (imirkin) [inline extract_firmware.py:424]: Why not convert this hunk as well?

**Review by imirkin — COMMENTED** [review]
Overall looks good, but:

**Comment by lgbaldoni** [comment]
>  Why not convert this hunk as well?

Not sure what you mean, that's a deleted line.

**Comment by imirkin** [comment]
That's what I mean -- you deleted a hunk of code instead of converting it. (It may not have existed in the version you started from.)

**Comment by lgbaldoni** [comment]
@imirkin The conversion was made starting from https://github.com/imirkin/re-vp2/blob/master/extract_firmware.py which appears to differ from https://github.com/envytools/firmware/blob/master/extract_firmware.py .

So, which is the authoritative version?

**Comment by imirkin** [comment]
The only difference is that one hunk, I think...

**Comment by imirkin** [comment]
This has been done differently in master, allowing py2 to remain supported as well.

**Pull request closed without merging**